compair/api/answer.py

```diff
@@ -249,6 +251,10 @@ def post(self, course_uuid, assignment_uuid):
                 abort(400, title="Answer Not Submitted",
                     message="You are currently not in any group for this course. Please contact your instructor to be added to a group.")
 
+        # set submission date if answer is being submitted
+        if not answer.draft:
```

Comment:
`if not answer.draft and not answer.submission_date:`
since we only want it to be updated once on first submission

This should also be in the update method

Reply:
Ok sorry that wasn't clear. I thought it would be updated if it gets edited after. I'll make this change

Reply:
@andrew-gardener what do you mean by "this should also be in the update method"?

Reply:
you put the login only in the create method but not in the update method.

Reply:
add it in here https://github.com/ubc/compair/blob/686ccd813a18e003649a22394e3cd5a93cb7acf9/compair/api/answer.py#L366 as well